src/components/container/containerStyled.tsx

```diff
@@ -1,14 +1,92 @@
 import styled from 'styled-components'
-import BGImage from '../../images/bg.jpg'
 
-export const ContainerStyled = styled.div`
-    width: 100vw;
-    min-height: 100vh;
-    background: url(${BGImage});
-    background-size: cover;
+/** Направление контейнера */
+export type ContainerDirection =
+    | 'row'
+    | 'column'
+    | 'row-reverse'
+    | 'column-reverse'
+
+/** Выравнивание по основной оси */
+export type ContainerMineAxisAlign =
+    | 'start'
+    | 'center'
+    | 'end'
+    | 'between'
+    | 'around'
+    | 'evenly'
+
+/** Выравнивание по перекрёcной оси */
+export type ContainerCrossAxisAlign = 'start' | 'center' | 'end' | 'stretch'
+
+/** Выравнивание контента  */
+export type ContainerContentAlign = ContainerMineAxisAlign
+
+export const ContainerStyled = styled.div<{
+    direction?: ContainerDirection
+    mineAxisAlign?: ContainerMineAxisAlign
+    crossAxisAlign?: ContainerCrossAxisAlign
+    contentAlign?: ContainerContentAlign
+}>`
     display: flex;
-    flex-direction: column;
-    align-items: center;
-    align-content: center;
-    justify-content: center;
+    flex: 1;
+    ${(props) => {
+        switch (props.direction) {
+            case 'column':
+                return 'flex-direction: column;'
+            case 'column-reverse':
+                return 'flex-direction: column-reverse;'
+            case 'row-reverse':
+                return 'flex-direction: row-reverse;'
+            default:
+                return 'flex-direction: row;'
+        }
+    }}
```

Comment:
Достаточно просто
```suggestion
        flex-direction: ${props => props.direction};
```
В остальных местах так же

Reply:
Принял, мой затуп